.buildkite/pipelines/java-ea-check-new-build.yml

```diff
@@ -0,0 +1,7 @@
+steps:
+  - command: .buildkite/scripts/trigger-if-java-ea-new-build.sh
+    env:
+      RECENT_TIME_WINDOW: "24" # time window to consider a build as new in hours
+    agents:
+      image: "docker.elastic.co/ci-agent-images/eck-region/buildkite-agent:1.5"
```

Comment:
thats the version of that image we use in other places. We should look into automating the updates via renovate

Reply:
See https://github.com/elastic/elasticsearch/pull/134122 as follow up on this